src/MSBuild/MSBuild/Microsoft.Build.CommonTypes.xsd

```diff
@@ -1700,6 +1705,11 @@ elementFormDefault="qualified">
         </xs:annotation>
     </xs:element>
     <xs:element name="DocumentationFile" type="msb:StringPropertyType" substitutionGroup="msb:Property"/>
+    <xs:element name="EmitCompilerGeneratedFiles" type="msb:boolean" substitutionGroup="msb:Property">
+        <xs:annotation>
+            <xs:documentation><!-- _locID_text="EmitCompilerGeneratedFiles" _locComment="" -->Controls if source generated files will be saved.</xs:documentation>
```

Comment:
One of the docs uses _saved_ the other uses _stored_ but it's the same action. I think we should use the same term for both of these.

Reply:
Change stored to saved